src/components/Video/ObservedAnimals.vue

```diff
@@ -108,6 +121,18 @@ export default {
         string += timestring;
       }
       return string;
+    },
+    confirmTag: function (rowItem) {
+      const animal = rowItem.animal;
+      const event = rowItem.event;
+      const tag = {};
+      if (event === 'false positive') {
+        tag.event = 'false positive';
+      } else {
+        tag.animal = animal;
+      }
+      tag.confidence = 0.6;
```

Comment:
Is there any way to make this the same as the tag being confirmed? (is that available?)

Failing that, please use 0.85 here as that's the default confidence we use elsewhere now.